### PR TITLE
Jobergum/streaming mode

### PR DESCRIPTION
**Pull request by jobergum**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: vespa/templates/services.xml [hunk 1]
```diff
@@ -11,6 +11,9 @@
         {% if schemas %}
         <search></search>
         <document-api></document-api>
+        <document-processing></document-processing>
+        {% endif %}
+        {% if search_clusters %}
         {% endif %}
```
Comment (jonmv) [inline vespa/templates/services.xml:17]: Isn't this just empty?

File: vespa/application.py [hunk 2]
```diff
@@ -979,9 +1005,10 @@ async def update_data(
         fields: Dict,
         create: bool = False,
         namespace: str = None,
+        groupname:str=None,
         **kwargs
     ) -> VespaResponse:
-        path = self.app.get_document_v1_path(id=data_id, schema=schema, namespace=namespace)
+        path = self.app.get_document_v1_path(id=data_id, schema=schema, namespace=namespace, group=groupname)
```
Comment (jonmv) [inline vespa/application.py:1011]: Just string groups? I guess that's the most common use-case.

**Review by jonmv — APPROVED** [review]

File: vespa/application.py (hunk 2 above)
Comment (jobergum) [reply vespa/application.py:1011]: According to doc, group is the way https://docs.vespa.ai/en/streaming-search.html#using-streaming-search

**Review by jonmv — APPROVED** [review]

File: vespa/application.py (hunk 2 above)
Comment (jonmv) [reply vespa/application.py:1011]: 👍

**Pull request merged**